webrtc/rtcpeerconnection/setLocalDescription.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>RTCPeerConnection setLocalDescription tests</title>
+</head>
+<body>
+  <!-- These files are in place when executing on W3C. -->
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script type="text/javascript">
+  // tests that the icegatheringstatechange event is called after calling setLocalDescription.
```

Comment:
This comment is redundant with the test title. Please drop it (they're slightly differently worded, so it may leave people confused.)